examples/contract-transfer/lib.rs

```diff
@@ -39,8 +24,8 @@ pub mod give_me {
         /// # Errors
         ///
         /// - Panics in case the requested transfer exceeds the contract balance.
-        /// - Panics in case the requested transfer would have brought the
-        ///   contract balance below the subsistence threshold.
+        /// - Panics in case the requested transfer would have brought this
+        ///   contract's balance below the minimum balance.
```

Comment:
What balance is the `minimum balance` referring to in this case?

Reply:
The chains existential deposit, I clarified it in the comment.